src/commands/impls/redisearch.rs

```diff
@@ -319,6 +319,7 @@ fn gen_search_options(args: &mut Vec<Value>, options: FtSearchOptions) -> Result
   }
   if !options.params.is_empty() {
     args.push(static_val!(PARAMS));
+    // TODO: times 2 for the number of arguments, no the number of pairs
     args.push(options.params.len().try_into()?);
```

Comment:
I don’t have an environment to easily run the test suite so I was hoping for this first commit without the fix to witness a failing CI run, but it looks like CI is not enabled on PRs from forks. The fix below works for me:

```suggestion
    args.push((options.params.len() * 2).try_into()?);
```